datastore/serializer/json.py

```diff
@@ -3,6 +3,7 @@
 
 import datastore.abc
 
+__all__ = ("Serializer",)
```

Comment:
Lets expose PrettySerializer also

Reply:
Yeah, just forgot about that one when copying-pasting that line. :slightly_smiling_face: